java/src/main/java/ai/onnxruntime/OrtSession.java

```diff
@@ -679,6 +691,32 @@ public void registerCustomOpLibrary(String path) throws OrtException {
       customLibraryHandles.add(customHandle);
     }
 
+    /**
+     * Sets the value of a symbolic dimension. Fixed dimension computations may have more
+     * optimizations applied to them.
+     *
+     * @param dimensionName The name of the symbolic dimension.
+     * @param dimensionValue The value to set that dimension to.
+     * @throws OrtException If there was an error in native code.
+     */
+    public void setSymbolicDimensionValue(String dimensionName, long dimensionValue)
+        throws OrtException {
+      checkClosed();
+      addFreeDimensionOverrideByName(
+          OnnxRuntime.ortApiHandle, nativeHandle, dimensionName, dimensionValue);
+    }
+
+    /**
+     * Disables the per session thread pools. Must be used in conjunction with an environment
+     * containing global thread pools.
+     *
+     * @throws OrtException If there was an error in native code.
+     */
+    public void disablePerSessionThreads() throws OrtException {
```

Comment:
Wouldn't exposing this create confusion since we don't have full support for global threadpools in Java API?

Reply:
I'm going to add that support once this PR has integrated, but I figure the other one will be kinda large and this is something that's easy to separate out.